### PR TITLE
created a model for points of interest, point category, squashed commits

### PR DESCRIPTION
**Pull request by ndmytro**

### CODE REVIEW

File: SSP/carcassonne/models.py [hunk 1]
```diff
@@ -60,3 +60,36 @@ class Workplace(models.Model):
...
     def __str__(self):
         return "Workplace {}".format(self.number)
+
+
+class PointCategory(models.Model):
+    """
+    Store data concerning point of interest categories.
+
+    Has attributes: category_name, category_image.
+    Related to model: 'carcassonne.PointCategory'
```
Comment (Fedorof) [inline SSP/carcassonne/models.py]: Could you please explain what this line means? It's a little obscuring for me.

File: SSP/carcassonne/tests/test_models.py [hunk 2]
```diff
@@ -120,6 +124,48 @@ def test_workplace_str(self):
...
         self.assertEqual(str(workplace), 'Workplace 1')
 
+@override_settings(
+    MEDIA_ROOT=os.path.join(BASE_DIR, "carcassonne/tests/tmp_dir"))
+class PointOfInterestTestCase(TestCase):
+    def setUp(self):
+        """Create PointCategory object"""
+        filename = os.path.join(BASE_DIR, 'carcassonne/tests/static/linux.jpg')
+        self._test_dir_path = os.path.join(BASE_DIR, "carcassonne/tests/tmp_dir")
+        with File(open(filename, 'rb')) as f:
```
Comment (Fedorof) [inline SSP/carcassonne/tests/test_models.py]: Why do you use `File`

File: SSP/carcassonne/tests/test_models.py [hunk 3]
```diff
@@ -120,6 +124,48 @@ def test_workplace_str(self):
...
+        self.assertEqual(str(pc), "Category name: Kitchen")
+
+    def testPointOfInterest(self):
+        """Point of interest has correct string representation."""
+        poi = Point.objects.get(name="Super Kitchen")
+        self.assertEqual(
+            str(poi),
+            "Point of interest, Category name: Kitchen, Floor 1 ")
+
+    def tearDown(self):
```
Comment (Fedorof) [inline SSP/carcassonne/tests/test_models.py]: Please, move `tearDown` right after `setUp`

File: SSP/carcassonne/models.py (hunk 1 above)
Comment (ndmytro) [inline SSP/carcassonne/models.py]: This line also looks weird to me, but I saw someone using such description in docstring and you didn't mind. I should definitely remove this line, shouldn't I?

File: SSP/carcassonne/tests/test_models.py (hunk 3 above)
Comment (ndmytro) [inline SSP/carcassonne/tests/test_models.py]: Will comply.

File: SSP/carcassonne/models.py (hunk 1 above)
Comment (Fedorof) [inline SSP/carcassonne/models.py]: Definitely.
BTW, the explanation sounds like a cargo cult

File: SSP/carcassonne/tests/test_models.py (hunk 2 above)
Comment (ndmytro) [inline SSP/carcassonne/tests/test_models.py]: This is also an implementation of cargo-cult=)

**Pull request merged**